.github/workflows/action.yml

```diff
@@ -0,0 +1,44 @@
+name: Satellite-Upgrade
+
+on:
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
+    steps:
+      - name: Checkout satellite6-upgrade
+        uses: actions/checkout@v2
+
+      - name: Set Up Python-${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install -r requirements-optional.txt
+
+      - name: Pre Commit Checks
+        uses: pre-commit/action@v2.0.0
+
+      - name: Upgrade Test Coverage (Existence Test)
+        run: py.test --cov=upgrade_tests/helpers upgrade_tests/helpers/coverme.py
+
+      - name: Make Docs
+        run: |
+          make test-docstrings
+          make docs
+      - name: Analysis (git diff)
+        if: failure()
+        run: git diff
+      - name: Upload Codecov Coverage
```

Comment:
I think we no more need Coverage now! We are not making use of it anyways. There has been a talk to entirely remove the CodeCoverage from Robottelo as well.

Reply:
Fixed.